src/vdaf/prio3.rs

```diff
@@ -28,10 +28,15 @@ use std::io::Cursor;
 use std::iter::IntoIterator;
 use std::marker::PhantomData;
 
-// TODO Add domain separation tag to field element generation as required by spec.
-
 // TODO Add test vectors and make sure they pass.
 
+// Domain-separation tag used to bind the VDAF operations to the document version. This will be
+// reved with each new draft.
+//
+// NOTE The CFRG has not yet adopted this spec. Version "vdaf-00" will match
+// draft-irtf-cfrg-vdaf-00.
+const VERS_PRIO3: &[u8] = b"vdaf-00 prio3";
```

Comment:
I like the idea of domain separation based on VDAF spec version. Does PPM need to deal with migration from one VDAF version to the next? Or do we claim that the VDAF version is an essential part of what VDAF is being executed, and thus in order to go from running `Prio3Count@vdaf-00` to `Prio3Count@vdaf-01`, you should deploy a completely new PPM task? In any case this question isn't germane to this PR?

Reply:
What will end up happening is that a particular draft of PPM will end up using a particular draft of VDAF.